_content/devices/index.md

```diff
@@ -17,7 +17,7 @@ Before your device can communicate via The Things Network, you will need to [reg
 
 ## Hardware
 
-Currently, The Things Network supports [LoRaWAN](https://www.lora-alliance.org/) for long range (~5 to 15km), low power (months to years on battery), but also low bandwidth (51 bytes/message) communication. We plan to support [Bluetooth LE](https://www.bluetooth.com/develop-with-bluetooth/build) and other networks as well.
+Currently, The Things Network supports [LoRaWAN](https://www.lora-alliance.org/) for long range (~5 to 15km), low power (months to years on battery), but also low bandwidth (51 bytes/message) communication. We plan to support [Bluetooth Mesh](https://blog.bluetooth.com/introducing-bluetooth-mesh-networking) and other networks as well.
```

Comment:
Mesh is too specific, although it's a viable option. It may be as simple as catching advertisement payload from sensors.

Reply:
@johanstokking what would you think of linking to [Bluetooth Broadcast/Mesh networks](https://www.bluetooth.com/bluetooth-technology/topology-options)

`[Bluetooth Broadcast/Mesh networks](https://www.bluetooth.com/bluetooth-technology/topology-options)`

Reply:
@MaartenArits can you use this link?

Reply:
Hi, yes i can :) I would go for that one as well. It was not clear to me you wanted to support all these typologies.